pyomo/contrib/benders/benders_cuts.py

```diff
@@ -0,0 +1,268 @@
+from pyomo.core.base.block import _BlockData, declare_custom_block
+import pyomo.environ as pe
+from pyomo.solvers.plugins.solvers.persistent_solver import PersistentSolver
+try:
+    from mpi4py import MPI
+    mpi4py_available = True
+except:
+    mpi4py_available = False
+import numpy as np
```

Comment:
You should add an import/availability check here since numpy is not a hard requirement of Pyomo